### PR TITLE
Regenerate gem from URL

### PR DESCRIPTION
**Pull request by atomaka**
```
bin/generate-from-hash 62735a73f63cf10085bc6077
```

where `62735a73f63cf10085bc6077` was retrieved from https://dev.synctera.com/reference/need-to-know in the download URL for the OpenAPI spec.

Note that this hash is the exact same one from last time :trollface:

### CODE REVIEW

**Review by Nroulston — APPROVED** [review]

File: docs/PersonsApi.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ All URIs are relative to *https://api-sandbox.synctera.com/v0*
 
 Create a person
 
-Create a person who may act as a personal customer or a director/officer/owner of a business. You can then verify the identity of this customer and associate them with other people and accounts. 
+Create a person who may act as a personal customer or a director/officer/owner of a business. You can then verify the identity of this customer and associate them with other people and accounts. Note that if no shipping_address attribute is provided in the request, the shipping_address will be set to a copy of the legal_address.
```
Comment (atomaka) [inline docs/PersonsApi.md:22]: TODO

File: docs/TransactionsApi.md [hunk 2]
```diff
@@ -251,7 +251,7 @@ opts = {
   account_no: 'account_no_example', # String | Account number
   exclude_jit_transactions: true, # Boolean | Hide \"JIT funding\" transactions from results
   page_token: 'a8937a0d', # String | 
-  account_id: '7d943c51-e4ff-4e57-9558-08cab6b963c7', # String | Account ID
+  account_id: ['inner_example'], # Array<String> | Account ID
```
Comment (atomaka) [inline docs/TransactionsApi.md:254]: TODO

File: docs/WebhookRequestObject.md [hunk 3]
```diff
@@ -9,6 +9,7 @@
 | **event_time** | **Time** | Timestamp of the current event raised |  |
 | **id** | **String** | The unique ID of the current event |  |
 | **metadata** | **String** | Metadata that stored in the webhook subscription |  |
+| **resource_id** | **String** | The ID of the resource this event occurred on, if available. | [optional][readonly] |
```
Comment (atomaka) [inline docs/WebhookRequestObject.md:12]: TODO

File: docs/WireReturnSimulationRequest.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Synctera::WireReturnSimulationRequest
```
Comment (atomaka) [inline docs/WireReturnSimulationRequest.md:1]: TODO

**Pull request merged**